syntax = "proto3";

package assign4;

service Database {
    rpc AccessDB (Request) returns (Response) {}
}

message Request {
    string req = 1;
}

message Response {
    string res = 1;
}
